### PR TITLE
Code reorganization

### LINKED ISSUE #166: Configure project to use yarn workspaces
We already have a number of stand-alone packages in the `lambdas` directory. There are also tests there, but it does not appear that those tests are being run. Can be modeled after https://github.com/mmazzarolo/tangerine-monorepo

I've used that tangerine-monorepo approach before, and it does make development faster an easier.

### PR DESCRIPTION
**Pull request by marchuffnagle**
- Get rid of sub-packges, there's no need for this project to be a monorepo at this point
- Move all tests in to a `tests` directory
- Add a `tests/README.md` that describes the different types of tests

Closes #166

### CODE REVIEW

File: .github/workflows/push.yaml [hunk 1]
```diff
@@ -36,5 +36,5 @@ jobs:
         run: npm audit --omit dev
       - name: Unit tests
         run: npm run test:coverage
-      - name: Integration tests
-        run: npm run test:integration
+      - name: System tests
```
Comment (philvarner) [inline .github/workflows/push.yaml:39]: 👍🏻

File: package.json [hunk 2]
```diff
@@ -34,7 +30,7 @@
   },
   "dependencies": {
     "@acuris/aws-es-connection": "^1.1.0",
-    "@elastic/elasticsearch": "^7.10",
+    "@elastic/elasticsearch": "^7.9.0",
```
Comment (philvarner) [inline package.json:33]: what was the problem here with 7.10?

Comment (marchuffnagle) [reply package.json:33]: The `@acuris/aws-es-connection` package only supports `@elastic/elasticsearch` up to version 7.9.0.

https://github.com/mergermarket/acuris-aws-es-connection/blob/3334c73a96db731970933c3240f1059c41301e50/package.json#L36

The 2.x.x version of `@acuris/aws-es-connection` supports later versions, but that contains breaking changes. For now we should be okay using 7.9

Comment (philvarner) [reply package.json:33]: ah, good to know

**Review by philvarner — APPROVED** [review]

**Pull request merged**